java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -615,23 +619,6 @@ protected <V> Map<GlideString, V> handleBinaryStringMapOrNullResponse(Response r
         return handleRedisResponse(Map.class, EnumSet.of(ResponseFlags.IS_NULLABLE), response);
     }
 
-    /**
-     * @param response A Protobuf response
-     * @return A map of a map of <code>String[][]</code>
-     */
-    protected Map<String, Map<String, String[][]>> handleXReadResponse(Response response)
```

Comment:
either let https://github.com/aws/glide-for-redis/pull/1784/files#diff-835b3496f82a26149f468dc3eca023d51623401f7c63e59b9cad3feef4f5a363R640
know that this is being moved (so they can move the binary version) when they rebase.  

Or don't move this.